tests/TestCase/Shell/CompletionShellTest.php

```diff
@@ -190,17 +193,32 @@ public function testSubCommandsAppPlugin()
         $this->Shell->runCommand(['subcommands', 'app.sample']);
         $output = $this->out->output;
 
-        $expected = '';
+        $expected = "derp\n";
         $this->assertEquals($expected, $output);
```

Comment:
If you use `assertTextEquals()` and remove the `\n` in the previous line, it will fix tests in windows